cmake/std/atdm/mutrino/environment.sh

```diff
@@ -10,7 +10,22 @@ if [ "$ATDM_CONFIG_COMPILER" == "DEFAULT" ] ; then
   export ATDM_CONFIG_COMPILER=INTEL
 fi
 
+if [ "$ATDM_CONFIG_KOKKOS_ARCH" == "DEFAULT" ] ; then
+  export ATDM_CONFIG_KOKKOS_ARCH=HSW
+fi
+
+if [ "$ATDM_CONFIG_KOKKOS_ARCH" != "HSW" ] && [ "$ATDM_CONFIG_KOKKOS_ARCH" != "KNL" ]; then
+  echo "***"
+  echo "*** ERROR: KOKKOS_ARCH=$ATDM_CONFIG_KOKKOS_ARCH is not a valid option on this system."
+  echo "*** '$ATDM_CONFIG_KOKKOS_ARCH' appears in $JOB_NAME which then sets the KOKKOS_ARCH."
+  echo "*** On Mutrino 'HSW' and 'KNL' are the only valid KOKKOS_ARCH settings."
+  echo "*** If no KOKKOS_ARCH is specified then 'HSW' will be used by default."
+  echo "***"
+  return
+fi
+
 echo "Using mutrino compiler stack $ATDM_CONFIG_COMPILER to build $ATDM_CONFIG_BUILD_TYPE code with Kokkos node type $ATDM_CONFIG_NODE_TYPE"
+echo "KOKKOS_ARCH=$ATDM_CONFIG_KOKKOS_ARCH"
```

Comment:
@fryeguy52, can we add `KOKKOS_ARCH` info to the above 'echo' statement?   It would be something like:

```
echo "Using mutrino compiler stack $ATDM_CONFIG_COMPILER to build $ATDM_CONFIG_BUILD_TYPE code with Kokkos node type $ATDM_CONFIG_NODE_TYPE and KOKKOS_ARCH $ATDM_CONFIG_KOKKOS_ARCH"
```